### ISSUE TITLE
missing createBuild

### ISSUE THREAD

**Issue by szymon-wesolowski** [microsoft/azure-devops-extension-api#31]
Missing createBuild function in BuildRestClient.

**Comment by pauldev263** [comment]
I totally agree.  My workaround was to locally override the Build interface in Build.d.ts to make all the parameters optional (added a ? suffix to all their names).  I was able to build solution locally supplying a minimal number of parameters for queueBuild.  Of course, whenever I check in changes, the build fails in AzDO as it pulls in the latest version of the api.

**Comment by Dejulia489** [comment]
I am also unable to queue a build. Can someone please provide an example?
